src/main/java/za/co/neslotech/spring/trainspringbootinsight/controller/UserController.java

```diff
@@ -0,0 +1,25 @@
+package za.co.neslotech.spring.trainspringbootinsight.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import za.co.neslotech.spring.trainspringbootinsight.model.User;
+import za.co.neslotech.spring.trainspringbootinsight.service.CrudService;
+import za.co.neslotech.spring.trainspringbootinsight.service.UserService;
+
+@RestController
+@RequestMapping("/api/v1/users")
+public class UserController extends CrudController<User, Long> {
+
+    private final UserService service;
+
+    @Autowired
```

Comment:
With the new version of spring, marking the constructor of a controller, service or repository annotated bean is redundant.